sdk/eventgrid/Azure.ResourceManager.EventGrid/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.3 (Unreleased)
```

Comment:
Please remove the section of `1.1.0-beta.3`, the latest version should be `1.1.0-beta.2`, that's why CI failed.